cmd/server/main.go

```diff
@@ -1,69 +1,44 @@
 package main
 
 import (
-	"fmt"
 	"net/http"
 	"os"
 
 	"log"
 
-	"github.com/golang-migrate/migrate/v4"
+	"github.com/broadinstitute/sherlock/internal/db"
+	"github.com/broadinstitute/sherlock/internal/sherlock"
+	"github.com/broadinstitute/sherlock/internal/tools"
 	_ "github.com/golang-migrate/migrate/v4/database/postgres"
 	_ "github.com/golang-migrate/migrate/v4/source/file"
 	_ "github.com/jackc/pgx/v4/stdlib"
```

Comment:
This is a DB driver, right? Is it a problem that we're using a different driver here versus in the migrations file?

Reply:
Whoops good catch! I was originally using the same driver for both then noticed it was deprecated in favor of `pgx` which is what was used here. Forgot to update the migrations though, will fix.

Reply:
fixed!